README.md

```diff
@@ -1,80 +1,190 @@
+
+##### Registering a route
+Tweeteor requires two routes. One root route that shows posts and one route for the profile page. Open up *router.js* and add these two routes. The profile route should use a parameter that is unique for each user, the username for instance.
+
+Create a data function for both routes that for now just return a predefined set of data so that you can verify that your routes work.
+
+*Tip: Iron router looks for templates named the same as you routes*
+
+*Tip: Look at the templates provided for posts and profile. Add some temporary code here to display the dummy data you put in the data function of your routes*
+
+##### Layout
+Right now, your app probably doesn't look to good. Let's fix this by using the layout we have provided. If you open *layout.html* you can see that this file has some default markup. 
+
+To use this layout in your router, open *router.js* and create a *Router.configure* function that takes in an object. Add a *layoutTemplate* property on this object with a string that matches the name of the layout template in *layout.html*. 
+
+To wrap it all up you need to tell the router where to *yield* the template rendered in the route function. Adding the special template function {{> yield}} does the trick.
+
+##### Handling data from subscriptions
+In later tasks you will add some collections to the application that you will publish from the server. The client will need to subscribe to these. But, since Tweeteor uses a router, this has to be handled a little differently than in the [chat application](https://github.com/EirikV/meteor-workshop/tree/chat-solution).
+
+Both the *route* function and the global *Router.configure* function supports an object property called *waitOn*. This is a function that can be used to set up route specific subscriptions (when used in a *route* call) and global subscriptions (when used in the *Router.configure* function). Either return a single *Meteor.subscribe* or an array of multiple subscriptions. If you want, you can add a loadingTemplate that will be displayed until the subscription is ready. Subscriptions aren't needed right now, but we will remind you when the time comes.
+
+##### 404 - header not found
+Iron router can of course show a custom 404 page for unknown routes. All you need to do is to use your *Router.configure* function to tell it where to find the *notFoundTemplate*. You can also show a custom *dataNotFound* page through the use of a *plugin*. Make your app a little more friendly by adding both of these.
+
+#### 3. Accounts
+A true social microblog needs to have authenticated users. Luckily, you don't have to build this functionality yourself. Meteor has a series of packages with the prefix *accounts* that handle user registration and authentication for you. With just a few keystrokes you can get authentication through Facebook, Twitter, Github and others.
```

Comment:
A true social microblog should have authenticated users that are able to register and log in.